src/DevBetterWeb.Infrastructure/Services/AlumniGraduationService.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DevBetterWeb.Core;
+using DevBetterWeb.Core.Entities;
+using DevBetterWeb.Core.Interfaces;
+using DevBetterWeb.Infrastructure.Identity.Data;
+using Microsoft.AspNetCore.Identity;
+
+namespace DevBetterWeb.Infrastructure.Services
+{
+  public class AlumniGraduationService : IAlumniGraduationService
+  {
+    private const int DAYS_IN_TWO_YEARS = 365 * 2;
+
```

Comment:
I like using this to make the final number clear.

Reply:
:)